From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Wed, 16 Nov 2022 11:18:07 +0100
Subject: Disable StartSurface feature

This feature would normally show a search box but leads to crashes.

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 chrome/browser/flags/android/chrome_feature_list.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/chrome/browser/flags/android/chrome_feature_list.cc b/chrome/browser/flags/android/chrome_feature_list.cc
--- a/chrome/browser/flags/android/chrome_feature_list.cc
+++ b/chrome/browser/flags/android/chrome_feature_list.cc
@@ -936,8 +936,8 @@ const base::Feature kTrustedWebActivityQualityEnforcementWarning{
 const base::Feature kShowExtendedPreloadingSetting{
     "ShowExtendedPreloadingSetting", base::FEATURE_ENABLED_BY_DEFAULT};
 
-const base::Feature kStartSurfaceAndroid{"StartSurfaceAndroid",
-                                         base::FEATURE_ENABLED_BY_DEFAULT};
+const base::Feature kStartSurfaceAndroid{"StartSurfaceAndroid",                // must be disabled
+                                         base::FEATURE_DISABLED_BY_DEFAULT};   // in Bromite
 
 const base::Feature kFeedPositionAndroid{"FeedPositionAndroid",
                                          base::FEATURE_DISABLED_BY_DEFAULT};
--
2.25.1
